main.py

```diff
@@ -553,11 +553,59 @@ def print_all_stats(commits_stats: dict, lines_stats: dict, contributors_stats:
 
 
 def main():
-    use_cloc = input("Do you want to use cloc (C) or wc (W) to count SLOC? c/W ").lower() == "c"
-    get_commits = input("Do you want to get the commits stats? It may take a long time due to GitHub servers updating "
-                        "their cache. y/N ").lower() == "y"
-    get_lines = input("Do you want to get the SLOC stats? It may take a long time since it has to clone each repository. y/N ").lower() == "y"
-    generate_graphs = input("Do you want to generate graphs for the statistics? y/N ").lower() == 'y'
```

Comment:
I believe these lines should be kept, but only run when no options are received.

Reply:
They already are kept, I moved them within an if/else to only execute them if no corresponding argument has been received